workers/go/workflow_utils/workflow_utils.go

```diff
@@ -0,0 +1,32 @@
+package workflow_utils
```

Comment:
```suggestion
package workflowutils
```

(and change dir, if we have to keep this package)